lib/templates/erb/scaffold/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<%# frozen_string_literal: true %>
+<%%= simple_form_for(@<%= singular_table_name %>) do |f| %>
```

Comment:
TIL about `<%%` and `<%%=` https://stackoverflow.com/a/40487347/638966

Reply:
Actually, let's just remove this. We don't really use scaffolding.